types/jsum/index.d.ts

```diff
@@ -0,0 +1,27 @@
+// Type definitions for jsum 0.1
+// Project: https://github.com/fraunhoferfokus/JSum#readme
+// Definitions by: David Uzumeri <https://github.com/daviduzumeri>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export as namespace jsum;
```

Comment:
Is the underlying library usable without an import?

Reply:
It is; you're right, this line was unnecessary.